### PR TITLE
Scrub Office 365

### PR DESCRIPTION
**Pull request by lindalu-MSFT**

### CODE REVIEW

**Comment by lindalu-MSFT** [comment]
@ElizabethSamuel-MSFT Please review brand scrub per your request. See bug https://office.visualstudio.com/OC/_workitems/edit/4857723

File: docs/design/add-in-icons-monoline.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ---
 title: Monoline style icon guidelines for Office Add-ins
 description: 'Get guidelines for using Monoline style icon icons in Office Add-ins.'
-ms.date: 12/09/2019
+ms.date: 2/09/2021
```
Comment (ElizabethSamuel-MSFT) [inline docs/design/add-in-icons-monoline.md:4]:
```suggestion
ms.date: 02/09/2021
```

File: docs/design/authentication-patterns.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ Add-ins may require users to sign-in or sign-up in order to access features and
...
+> The single sign-on API is currently supported for Word, Excel, Outlook, and PowerPoint. For more information about single sign-on support, see [IdentityAPI requirement sets](../reference/requirement-sets/identity-api-requirement-sets.md). If you are working with an Outlook add-in, be sure to enable Modern Authentication for the Office tenancy. For information about how to do this, see [Exchange Online: How to enable your tenant for modern authentication](https://social.technet.microsoft.com/wiki/contents/articles/32711.exchange-online-how-to-enable-your-tenant-for-modern-authentication.aspx).
```
Comment (ElizabethSamuel-MSFT) [inline docs/design/authentication-patterns.md:48]: Wouldn't tenancy be an aspect of M365 subscription?
```suggestion
> The single sign-on API is currently supported for Word, Excel, Outlook, and PowerPoint. For more information about single sign-on support, see [IdentityAPI requirement sets](../reference/requirement-sets/identity-api-requirement-sets.md). If you are working with an Outlook add-in, be sure to enable Modern Authentication for the Microsoft 365 tenancy. For information about how to do this, see [Exchange Online: How to enable your tenant for modern authentication](https://social.technet.microsoft.com/wiki/contents/articles/32711.exchange-online-how-to-enable-your-tenant-for-modern-authentication.aspx).
```

File: docs/design/office-ui-fabric.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
 ---
 title: Office UI Fabric in Office Add-ins 
 description: 'Get an overview of how to use the Office UI Fabric components in Office Add-ins.'
-ms.date: 10/29/2020
+ms.date: 2/09/2021
```
Comment (ElizabethSamuel-MSFT) [inline docs/design/office-ui-fabric.md:4]:
```suggestion
ms.date: 02/09/2021
```

File: docs/develop/authorize-to-microsoft-graph.md [hunk 4]
```diff
@@ -11,7 +11,7 @@ Users sign in to Office (online, mobile, and desktop platforms) using either the
 
 > [!NOTE]
 > The Single Sign-on API is currently supported for Word, Excel, Outlook, and PowerPoint. For more information about where the Single Sign-on API is currently supported, see [IdentityAPI requirement sets](../reference/requirement-sets/identity-api-requirement-sets.md).
-> If you are working with an Outlook add-in, be sure to enable Modern Authentication for the Office 365 tenancy. For information about how to do this, see [Exchange Online: How to enable your tenant for modern authentication](https://social.technet.microsoft.com/wiki/contents/articles/32711.exchange-online-how-to-enable-your-tenant-for-modern-authentication.aspx).
+> If you are working with an Outlook add-in, be sure to enable Modern Authentication for the Office tenancy. For information about how to do this, see [Exchange Online: How to enable your tenant for modern authentication](https://social.technet.microsoft.com/wiki/contents/articles/32711.exchange-online-how-to-enable-your-tenant-for-modern-authentication.aspx).
```
Comment (ElizabethSamuel-MSFT) [inline docs/develop/authorize-to-microsoft-graph.md:14]:
```suggestion
> If you are working with an Outlook add-in, be sure to enable Modern Authentication for the Microsoft 365 tenancy. For information about how to do this, see [Exchange Online: How to enable your tenant for modern authentication](https://social.technet.microsoft.com/wiki/contents/articles/32711.exchange-online-how-to-enable-your-tenant-for-modern-authentication.aspx).
```

File: docs/develop/office-versions-and-requirement-sets.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Office versions and requirement sets
 description: 'Supported Office.js platforms using JavaScript API'
```
Comment (ElizabethSamuel-MSFT) [inline docs/develop/office-versions-and-requirement-sets.md:3]:
```suggestion
description: 'Supported Office.js platforms using JavaScript API.'

```

File: docs/develop/troubleshoot-sso-in-office-add-ins.md [hunk 6]
```diff
@@ -11,7 +11,7 @@ This article provides some guidance about how to troubleshoot problems with sing
 
 > [!NOTE]
 > The Single Sign-on API is currently supported for Word, Excel, Outlook, and PowerPoint. For more information about where the Single Sign-on API is currently supported, see [IdentityAPI requirement sets](../reference/requirement-sets/identity-api-requirement-sets.md).
-> If you are working with an Outlook add-in, be sure to enable Modern Authentication for the Office 365 tenancy. For information about how to do this, see [Exchange Online: How to enable your tenant for modern authentication](https://social.technet.microsoft.com/wiki/contents/articles/32711.exchange-online-how-to-enable-your-tenant-for-modern-authentication.aspx).
+> If you are working with an Outlook add-in, be sure to enable Modern Authentication for the Office tenancy. For information about how to do this, see [Exchange Online: How to enable your tenant for modern authentication](https://social.technet.microsoft.com/wiki/contents/articles/32711.exchange-online-how-to-enable-your-tenant-for-modern-authentication.aspx).
```
Comment (ElizabethSamuel-MSFT) [inline docs/develop/troubleshoot-sso-in-office-add-ins.md:14]:
```suggestion
> If you are working with an Outlook add-in, be sure to enable Modern Authentication for the Microsoft 365 tenancy. For information about how to do this, see [Exchange Online: How to enable your tenant for modern authentication](https://social.technet.microsoft.com/wiki/contents/articles/32711.exchange-online-how-to-enable-your-tenant-for-modern-authentication.aspx).
```

File: docs/outlook/add-in-requirements.md [hunk 7]
```diff
@@ -17,11 +17,11 @@ For Outlook add-ins to load and function properly, there are a number of require
    - Outlook 2016 or later on Mac
    - Outlook on iOS
    - Outlook on Android
-   - Outlook on the web for Exchange 2016 or later and Office 365
+   - Outlook on the web for Exchange 2016 or later
    - Outlook on the web for Exchange 2013
    - Outlook.com
 
-- The client must be connected to an Exchange server or Microsoft 365 using a direct connection. When configuring the client, the user must choose an **Exchange**, **Office 365**, or **Outlook.com** account type. If the client is configured to connect with POP3 or IMAP, add-ins will not load.
+- The client must be connected to an Exchange server or Microsoft 365 using a direct connection. When configuring the client, the user must choose an **Exchange**, **Office**, or **Outlook.com** account type. If the client is configured to connect with POP3 or IMAP, add-ins will not load.
```
Comment (ElizabethSamuel-MSFT) [inline docs/outlook/add-in-requirements.md:24]: Is there anyway to confirm what the current value is? Was it changed from "Office 365"?

File: docs/outlook/authentication.md [hunk 8]
```diff
@@ -15,7 +15,7 @@ Single sign-on access tokens provide a seamless way for your add-in to authentic
 
 > [!NOTE]
 > The Single Sign-on API is currently supported for Word, Excel, Outlook, and PowerPoint. For more information about where the Single Sign-on API is currently supported, see [IdentityAPI requirement sets](../reference/requirement-sets/identity-api-requirement-sets.md).
-> If you are working with an Outlook add-in, be sure to enable Modern Authentication for the Office 365 tenancy. For information about how to do this, see [Exchange Online: How to enable your tenant for modern authentication](https://social.technet.microsoft.com/wiki/contents/articles/32711.exchange-online-how-to-enable-your-tenant-for-modern-authentication.aspx).
+> If you are working with an Outlook add-in, be sure to enable Modern Authentication for the Office tenancy. For information about how to do this, see [Exchange Online: How to enable your tenant for modern authentication](https://social.technet.microsoft.com/wiki/contents/articles/32711.exchange-online-how-to-enable-your-tenant-for-modern-authentication.aspx).
```
Comment (ElizabethSamuel-MSFT) [inline docs/outlook/authentication.md:18]:
```suggestion
> If you are working with an Outlook add-in, be sure to enable Modern Authentication for the Microsoft 365 tenancy. For information about how to do this, see [Exchange Online: How to enable your tenant for modern authentication](https://social.technet.microsoft.com/wiki/contents/articles/32711.exchange-online-how-to-enable-your-tenant-for-modern-authentication.aspx).
```

File: docs/outlook/compose-scenario.md [hunk 9]
```diff
@@ -51,4 +51,4 @@ Compose add-ins are implemented as [Add-in commands for Outlook](add-in-commands
 
 ## See also
 
-- [Get Started with Outlook add-ins for Office 365](../quickstarts/outlook-quickstart.md)
+- [Get Started with Outlook add-ins for Office](../quickstarts/outlook-quickstart.md)
```
Comment (ElizabethSamuel-MSFT) [inline docs/outlook/compose-scenario.md:54]:
```suggestion
- [Get Started with Outlook add-ins](../quickstarts/outlook-quickstart.md)
```

File: docs/outlook/implement-sso-in-outlook-add-in.md [hunk 10]
```diff
@@ -14,7 +14,7 @@ For a sample add-in that implements the ideas in this article, see [Outlook Add-
 
 > [!NOTE]
 > The Single Sign-on API is currently supported for Word, Excel, Outlook, and PowerPoint. For more information about where the Single Sign-on API is currently supported, see [IdentityAPI requirement sets](../reference/requirement-sets/identity-api-requirement-sets.md).
-> If you are working with an Outlook add-in, be sure to enable Modern Authentication for the Office 365 tenancy. For information about how to do this, see [Exchange Online: How to enable your tenant for modern authentication](https://social.technet.microsoft.com/wiki/contents/articles/32711.exchange-online-how-to-enable-your-tenant-for-modern-authentication.aspx).
+> If you are working with an Outlook add-in, be sure to enable Modern Authentication for the Office tenancy. For information about how to do this, see [Exchange Online: How to enable your tenant for modern authentication](https://social.technet.microsoft.com/wiki/contents/articles/32711.exchange-online-how-to-enable-your-tenant-for-modern-authentication.aspx).
```
Comment (ElizabethSamuel-MSFT) [inline docs/outlook/implement-sso-in-outlook-add-in.md:17]:
```suggestion
> If you are working with an Outlook add-in, be sure to enable Modern Authentication for the Microsoft 365 tenancy. For information about how to do this, see [Exchange Online: How to enable your tenant for modern authentication](https://social.technet.microsoft.com/wiki/contents/articles/32711.exchange-online-how-to-enable-your-tenant-for-modern-authentication.aspx).
```

File: docs/outlook/sideload-outlook-add-ins-for-testing.md [hunk 11]
```diff
@@ -26,7 +26,7 @@ The process for sideloading an add-in in Outlook on the web depends upon whether
 
 ### Sideload an add-in in the new Outlook on the web
 
-1. Go to [Outlook in Office 365](https://outlook.office.com).
+1. Go to [https://outlook.office.com](https://outlook.office.com).
 
 1. In Outlook on the web, create a new message.
```
Comment (ElizabethSamuel-MSFT) [inline docs/outlook/sideload-outlook-add-ins-for-testing.md:31]:
```suggestion
1. Go to [Outlook on the web](https://outlook.office.com).

1. Create a new message.
```

File: docs/outlook/sideload-outlook-add-ins-for-testing.md [hunk 12]
```diff
@@ -46,7 +46,7 @@ The process for sideloading an add-in in Outlook on the web depends upon whether
 
 ### Sideload an add-in in classic Outlook on the web
 
-1. Go to [Outlook in Office 365](https://outlook.office.com).
+1. Go to [https://outlook.office.com](https://outlook.office.com).
```
Comment (ElizabethSamuel-MSFT) [inline docs/outlook/sideload-outlook-add-ins-for-testing.md:49]:
```suggestion
1. Go to [Outlook on the web](https://outlook.office.com).
```

File: docs/reference/manifest/webapplicationinfo.md [hunk 13]
```diff
@@ -13,7 +13,7 @@ Supports single sign-on (SSO) in Office Add-ins. This element contains informati
...
+> The single sign-on API is currently supported for Word, Excel, Outlook, and PowerPoint. For more information about where the single sign-on API is currently supported, see [IdentityAPI requirement sets](../requirement-sets/identity-api-requirement-sets.md). If you are working with an Outlook add-in, be sure to enable Modern Authentication for the Office tenancy. To learn how to do this, see [Exchange Online: How to enable your tenant for modern authentication](https://social.technet.microsoft.com/wiki/contents/articles/32711.exchange-online-how-to-enable-your-tenant-for-modern-authentication.aspx).
```
Comment (ElizabethSamuel-MSFT) [inline docs/reference/manifest/webapplicationinfo.md:16]:
```suggestion
> The single sign-on API is currently supported for Word, Excel, Outlook, and PowerPoint. For more information about where the single sign-on API is currently supported, see [IdentityAPI requirement sets](../requirement-sets/identity-api-requirement-sets.md). If you are working with an Outlook add-in, be sure to enable Modern Authentication for the Microsoft 365 tenancy. To learn how to do this, see [Exchange Online: How to enable your tenant for modern authentication](https://social.technet.microsoft.com/wiki/contents/articles/32711.exchange-online-how-to-enable-your-tenant-for-modern-authentication.aspx).
```

File: docs/reference/requirement-sets/open-browser-window-api-requirement-sets.md [hunk 14]
```diff
@@ -14,15 +14,15 @@ The OpenBrowserWindow API set enables add-ins to open a browser to accomplish ta
...
+|  Requirement set  | Office 2013 on Windows or later<br>(one-time purchase) | Office on Windows<br>(connected to Microsoft 365 subscription) |  Office on iPad<br>(connected to Microsoft 365 subscription)  |  Office on Mac<br>(connected to Microsoft 365 subscription)  | Office on the web  |  Office Online Server  |
 |:-----|-----|:-----|:-----|:-----|:-----|:-----|:-----|
 | OpenBrowserWindowApi 1.1  | N/A | Version 1810 (Build 16.0.11001.20074) or later | 16.0.0.0 or later | 16.0.0.0 or later | N/A | N/A|
 
 To find out more about versions, build numbers, and Office Online Server, see:
 
-- [Version and build numbers of update channel releases for Office 365 clients](https://support.office.com/article/version-and-build-numbers-of-update-channel-releases-ae942449-1fca-4484-898b-a933ea23def7)
+- [Version and build numbers of update channel releases for Office clients](https://support.office.com/article/version-and-build-numbers-of-update-channel-releases-ae942449-1fca-4484-898b-a933ea23def7)
```
Comment (ElizabethSamuel-MSFT) [inline docs/reference/requirement-sets/open-browser-window-api-requirement-sets.md:23]:
```suggestion
- [Version and build numbers of update channel releases for Microsoft 365 Apps](/officeupdates/update-history-microsoft365-apps-by-date)
```

File: docs/tutorials/migrate-vsto-to-office-add-in-shared-code-library-tutorial.md [hunk 15]
```diff
@@ -45,17 +45,17 @@ To set up your development environment:
 
 You also need the following:
 
-- A Microsoft 365 account. You can join the [Microsoft 365 developer program](https://aka.ms/devprogramsignup) that includes a free 1 year subscription to Office 365.
+- A Microsoft 365 account. You can join the [Microsoft 365 developer program](https://aka.ms/devprogramsignup) that includes a renewable 90 day subscription to Microsoft 365 that includes Office apps.
```
Comment (ElizabethSamuel-MSFT) [inline docs/tutorials/migrate-vsto-to-office-add-in-shared-code-library-tutorial.md:48]:
```suggestion
- A Microsoft 365 account. You can join the [Microsoft 365 developer program](https://aka.ms/devprogramsignup) that provides a renewable 90-day Microsoft 365 subscription that includes Office apps.
```

**Pull request merged**